### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Home_Text spacing: After applying text spacing the alignment of the content present in the "Home" page is not proper.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6198]
### GitHubTags
#WCAG1.4.12; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Text Spacing;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Pre-Requisites

1. Open https://www.html5accessibility.com/tests/tsbookmarklet.html
2. Drag link "Text Spacing" to bookmarklets bar in browser.
3. Apply text spacing.

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Verify whether the alignment of the content present in the "Home" page is proper or not.

### Actual Result
After applying text spacing the alignment of the content present in the "Home" page is not proper.

### Expected Result
After applying text spacing the alignment of the content present in the "Home" page should be proper.

### User Impact
Users/People with Dyslexia are impacted with text spacing issue.

### Attachment
![Bug_Text spacing](https://user-images.githubusercontent.com/99243079/229982718-e487aafa-a2b1-41af-8def-b071b56e40e8.gif)

**Comment by gregvanl** [comment]
Closing old issue as "won't fix"
